### ISSUE TITLE
Improve Demo to support Safari Browser

### ISSUE THREAD

**Issue by jpvajda** [deepgram-devs/js-live-example#1]
See related Discussion: https://github.com/orgs/deepgram/discussions/557

The "Demo: Transcribe your voice live" page works wonderfully in Chrome / Safari, but the linked repo
https://github.com/deepgram-devs/js-live-example only works in Chrome, and NOT Safari.

The issue seems similar to a couple other posts here and many others elsewhere: Safari doesn't support audio/webm.

But somehow Deepgram's own demo does it properly! So I wish I could just replicate it.

Specifically, in Safari using js-live-example the connection closes immediately once the first message is sent.

**Comment by dmxt** [comment]
I agree. It doesn't work on mobile phone both on Chrome or Safari. The button is unclickable. It stays on default state (blue, instead of red's listening mode) on click.
